resources/odvd/ODVDFH16Truck.odvd

```diff
@@ -16,11 +16,12 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-message opendlv.proxy.reverefh16.ManualControl [id = 191] {
+message opendlv.proxy.reverefh16.ManualDriver [id = 191] {
```

Comment:
As this PR contains renaming of messages, please make sure that you propagate this renaming to the higher-level repositories